packages/scandipwa/src/route/CartPage/CartPage.container.js

```diff
@@ -207,19 +206,7 @@ export class CartPageContainer extends PureComponent {
         changeHeaderState({
             name: CART,
             title,
-            onEditClick: () => {
```

Comment:
I assume we are not using edit  option on cartPage, at least I didn't saw any button or something like that on it.